### DISCUSSION TITLE
fswebcam and config file

### DISCUSSION THREAD

**Discussion by aproxxo** [kizniche/Mycodo#1380] [General]
Hi,

My webcam (fswebcam) shows very bad pictures in Mycodo.
Now I have created a parameter file, webcam.cfg.

With the command "fswebcam -c webcam.cfg" the pictures are now perfect. How can I now integrate the parameter file into Mycodo?

Greeting

**Comment by kizniche** [comment]
The Custom Options field inserts text into the command that's executed.
